environments/auth-with-multi-alias/README.md

```diff
@@ -0,0 +1,25 @@
+# Auth with Multiple Sign In Aliases
+
+This environment allows users to sign in using a username, email, or phone number. Allowing users to sign in using multiple aliases requires configuring your `auth` backend using the Amplify CLI.
+
+## Configuring the Backend
+
+To configure your application's `auth` backend to allow multiple sign in aliases, follow these steps:
+
+1. In your application's root directory, add authentication using the Amplify CLI:
+
+   ```shell
+   amplify add auth
+   ```
+
+1. When prompted `Do you want to use the default authentication and security configuration?`, select `Manual configuration`
+1. When prompted `How do you want users to be able to sign in?`, select the aliases you would like users to be able to use to sign in to your application
+   ![](multi-alias-config.png)
```

Comment:
The image is kinda inline here https://github.com/aws-amplify/amplify-ui/blob/26667e59f49853772c6db94e67cd3590acf68bfb/environments/auth-with-multi-alias/README.md, so let's give it some breathing room:

```suggestion
1. When prompted `How do you want users to be able to sign in?`, select the aliases you would like users to be able to use to sign in to your application:

    > ![](multi-alias-config.png)
```